Classes/Command/ExtensionCommandController.php

```diff
@@ -122,6 +125,18 @@ public function deactivateCommand(array $extensionKeys)
         }
     }
 
+    private function showDeprecationMessageIfApplicable()
+    {
+        if (Bootstrap::usesComposerClassLoading()) {
+            $this->output->outputLine('<warning>This command is deprecated when TYPO3 is composer managed.</warning>');
```

Comment:
I guess this could be simplified by [formatting as block](https://symfony.com/doc/current/components/console/helpers/formatterhelper.html#print-messages-in-a-block) or at least with `ConsoleOutput::outputFormatted()`.

Reply:
hm, good point, but `ConsoleOutput::outputFormatted` does not use `formatBlock`